packages/synapse-react-client/src/components/FeaturedResearch/FeaturedResearch.test.tsx

```diff
@@ -0,0 +1,192 @@
+            columnType: ColumnTypeEnum.DATE,
+            id: '3',
+          },
+          {
+            name: 'tags',
+            columnType: ColumnTypeEnum.STRING_LIST,
+            id: '4',
+          },
+          {
+            name: 'affiliation',
+            columnType: ColumnTypeEnum.STRING,
+            id: '5',
+          },
+          {
+            name: 'image',
+            columnType: ColumnTypeEnum.FILEHANDLEID,
+            id: '6',
+          },
+          { name: 'link', columnType: ColumnTypeEnum.LINK, id: '7' },
+          { name: 'order', columnType: ColumnTypeEnum.INTEGER, id: '8' },
+        ],
+        rows: [
+          {
+            rowId: 1,
+            values: [
+              'Title 1',
+              'Description 1',
+              '1726164997000',
+              '["tag1_1", "tag1_2"]',
+              'affiliation 1',
+              '149976034',
+              'https://mockurl.com/data-release-1',
+              '2',
+            ],
+          },
+          {
+            rowId: 2,
+            values: [
+              'Title 2',
+              'Description 2',
+              '1726164997000',
+              '["tag2_1"]',
+              'affiliation 2',
+              '149976044',
+              'https://mockurl.com/data-release-2',
+              '2',
+            ],
+          },
+        ],
+      },
+    },
+    selectColumns: [
+      {
+        name: 'title',
+        columnType: ColumnTypeEnum.STRING,
+        id: '1',
+      },
+      {
+        name: 'description',
+        columnType: ColumnTypeEnum.STRING,
+        id: '2',
+      },
+      {
+        name: 'publicationDate',
+        columnType: ColumnTypeEnum.DATE,
+        id: '3',
+      },
+      {
+        name: 'tags',
+        columnType: ColumnTypeEnum.STRING_LIST,
+        id: '4',
+      },
+      {
+        name: 'affiliation',
+        columnType: ColumnTypeEnum.STRING,
+        id: '5',
+      },
+      {
+        name: 'image',
+        columnType: ColumnTypeEnum.FILEHANDLEID,
+        id: '6',
+      },
+      { name: 'link', columnType: ColumnTypeEnum.LINK, id: '7' },
+      { name: 'order', columnType: ColumnTypeEnum.INTEGER, id: '8' },
+    ],
+  }
+
+  const mockFileResult = [
+    {
+      fileHandleId: '149976034',
+      preSignedURL: 'https://mockurl.com/orangecat.jpeg',
+    },
+    {
+      fileHandleId: '149976044',
+      preSignedURL: 'https://mockurl.com/tabbycat.jpeg',
+    },
+  ]
+
+  const mockBatchFileResult: BatchFileResult = {
+    requestedFiles: mockFileResult,
+  }
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    jest.spyOn(SynapseClient, 'getFiles').mockResolvedValue(mockBatchFileResult)
+    mockUseGetQueryResultBundle.mockReturnValue(
+      getUseQuerySuccessMock(mockQueryResult),
+    )
+  })
+
+  const renderWithRouter = (props: FeaturedResearchProps) => {
+    const router = createMemoryRouter([
+      {
+        path: '/',
+        element: <FeaturedResearch {...props} />,
+      },
+    ])
```

Comment:
Seems like the component is not using react-router, so you probably don't need to use it in tests